package.json

```diff
@@ -45,13 +45,14 @@
   },
   "browserslist": "Node 12.20 - 13 and Node < 13, Node >= 14.13, > 0.5%, not OperaMini all, not IE > 0, not dead",
   "peerDependencies": {
+    "@apollo/client": "^3",
```

Comment:
We need to figure out and define exactly the minimum version of the `@apollo/client` peer dependency that's compatible - from memory there were issues in the early v3 releases.

Reply:
Ok, so `@apollo/client` v3.0.0 is definitely not compatible, because we `require` `ApolloLink` and `Observable` from `@apollo/client/core`, and that's not exported from there in that version:

https://github.com/jaydenseric/apollo-upload-client/blob/81dc6dd2ef517a88b0151a701d4bd3515a022836/public/createUploadLink.js#L3

https://unpkg.com/browse/@apollo/client@3.0.0/core/core.cjs.js

Reply:
Hmm, maybe I was mistaken. I did a test `@apollo/client` v3.0.0 installation and required all the things we need and it does seem to export everything.